README.md

```diff
@@ -27,8 +37,290 @@ pip install -e .
+pixel (hot, cosmic, etc.). Instrument pipelines generally come with bad pixels
+map generator, but you can add an unlimited number of bad pixels as the input
+list `add_bad` (e.g.: \[[24, 08], [31, 41]])).
+
+```python
+nrm_file = 'my_nrm_data.fits'
+
+clean_param = {'isz': 69, # final cropped image size [pix]
+               'r1': 31, # Inner radius to compute sky [pix]
+               'dr': 2, # Outer radius (r2 = r1 + dr) 
+               'apod': True, # If True, apply windowing
+               'window': 32 # FWHM of the super-gaussian (for windowing)
+              }
+
+# Firsly, check if the input parameters are valid
+amical.check_data_params(nrm_file, **clean_param)
+```
+
+<p align="center">
+<img src="Figures/cleaning_params.png" width="50%"/>
+</p>
+
+If the cleaning parameters seem well located (cyan cross on the centre, sky
+radius outside the fringes pattern, etc.), we can apply the cleaning step to the
+data.
+
+```python
+
+cube_cleaned = amical.select_clean_data(nrm_file, **clean_param, clip=True, clip_fact=0.5)
+```
+
+During the cleaning step, you can decide to apply a lucky imaging selection
+(`clip`=True) to accumulate only the best frames in the cube (based on the
+integrated fluxes compared to the median: threshold = median(fluxes) -
+`clip_fact` x std(fluxes)).
+
+<p align="center">
+<img src="Figures/clipping.png" width="80%"/>
+</p>
+
+### Step 2: extract observables
+
+The second step is the core of AMICAL: we use the Fourier sampling approach to
+extract the interferometric observables (visibilities and closure phases).
+
+All the challenge when you play with the NRM data is to find the correct
+position of each baseline in the Fourier transform. To do so, we implemented 4
+different sampling methods (`peakmethod` = ('unique', 'square', 'gauss', 'fft'))
+to exploit information spread beyond just the _u_, _v_ positions.
+
+<p align="center">
+<img src="Figures/fft.png" width="60%"/>
+</p>
+<p align="center">
+<img src="Figures/peakmethod.png" width="80%"/>
+</p>
+
+Based on NIRISS and SPHERE dataset analysis, we recommend using the **fft**
+method (but feel free to test the other methods for your specific case!). The
+expected baseline locations on the detector are computed using the mask
+coordinates, the wavelength and the pixel size. In some cases, the mask is not
+perfectly aligned with the detector and so requires to be rotated
+(`theta_detector` = 0) or centrally scaled (`scaling_uv` = 1). 
+
+With AMICAL, the mask coordinates, the wavelengths, the pixel size and the
+target name are normally determined using the fits header informations.
+Otherwise, you will need to give `filtname`,`instrum` and `targetname` to
+determine those information from the AMICAL internal database
+([get_infos_obs.py](amical/get_infos_obs.py)).
+
+```python
+params_ami = {"peakmethod": "fft",
+              "maskname": "g7", # 7 holes mask of NIRISS
+              }
+
+bs = amical.extract_bs(cube_cleaned, file_t, **params_ami)
```

Comment:
Unless you mean to reuse this dictionary later in the tutorial, there is no need for it.